tests/tests_resize.yml

```
@@ -37,211 +37,217 @@
 
     # For ext4 FS
 
-    - name: >-
-        Create one LVM logical volume under one volume group with size
-        {{ volume_size_before }}
-      include_role:
-        name: linux-system-roles.storage
-      vars:
-        storage_pools:
-          - name: foo
-            disks: "{{ unused_disks }}"
-            type: lvm
-            volumes:
-              - name: test1
-                # resizing is currently supported only for ext2/3/4
-                fs_type: ext4
-                size: "{{ volume_size_before }}"
-                mount_point: "{{ mount_location }}"
-
-    - name: Verify role results
-      include_tasks: verify-role-results.yml
-
-    - name: Change volume_size to {{ volume_size_after }}
-      include_role:
-        name: linux-system-roles.storage
-      vars:
-        storage_pools:
-          - name: foo
-            type: lvm
-            disks: "{{ unused_disks }}"
-            volumes:
-              - name: test1
-                fs_type: 'ext4'
-                size: "{{ volume_size_after }}"
-                mount_point: "{{ mount_location }}"
+    - name: ext4 FS
```

```suggestion
    - name: Test ext4 FS
```
ansible-lint requires names to start with an upper case letter

---

Yup, already fixed locally. I just don't want to push more often than necessary, as the tests are so expensive (running on my fork ATM to avoid some noise)